### PR TITLE
feat: utlize async queue to wait for pending notify calls

### PR DESCRIPTION
**Pull request by capJavert**
We found an issue where async method `notifyGeneratePersonalizedDigest` does not get to be resolved before the stream ends and the instance that is running a cron destroys itself. 

Utlizes async queue to manage notify tasks and awaits for queue to be drained (completed) before exiting.

### CODE REVIEW

File: src/cron/personalizedDigest.ts [hunk 1]
```diff
@@ -14,11 +15,18 @@ const cron: Cron = {
       });
 
     const personalizedDigestStream = await personalizedDigestQuery.stream();
+    const notifyQueueConcurrency = 1000;
+    const notifyQueue = fastq.promise(
+      async (personalizedDigest: UserPersonalizedDigest) => {
+        await notifyGeneratePersonalizedDigest(logger, personalizedDigest);
+      },
```
Comment (idoshamun) [inline src/cron/personalizedDigest.ts:22]:
```suggestion
      (personalizedDigest: UserPersonalizedDigest) => notifyGeneratePersonalizedDigest(logger, personalizedDigest),
```

**Review by idoshamun — APPROVED** [review]
Amazing!

**Review by rebelchris — APPROVED** [review]
Nice nothing blocking from my side.
Let those emails come 👋

File: src/cron/personalizedDigest.ts (hunk 1 above)
Comment (capJavert) [reply src/cron/personalizedDigest.ts:22]: @idoshamun for some reason with above changes tests timed out, not sure if it's a jest thing, fastq thing https://app.circleci.com/pipelines/github/dailydotdev/daily-api/5895/workflows/db8d60e6-58fc-422b-ac89-dc8f4f0bc8fa/jobs/23036 cc: @rebelchris any clue?

Reverted below and tests passed again.

**Pull request merged**